test-lit/driver/version_test.f90

```diff
@@ -1,5 +1,7 @@
 ! Check that lit configuration works by checking the compiler version
 
 ! RUN: %f18 -V 2>&1 | FileCheck  -check-prefix=VERSION %s
+! VERSION: {{[[:space:]]}}
```

Comment:
This test will still pass with the following incorrect output from `f18 -V`:

> This is a bug
> 
> This is also a bug
> 
> f18 compiler (under development) This too
> 
> And this

Reply:
The task to check if there is nothing else before the expected text is not trivial with FileCheck.
These discussions  show how to implement a check of space with regression tests.
https://reviews.llvm.org/D28896 https://reviews.llvm.org/D29122
There is also the command CHECK-EMPTY but it cannot be used in the beginning of the checks
The task is also difficult to check the end of the file for nothing ever.

Reply:
For the benefit of the review, Carol and I had a face-to-face conversation about this and agreed that as it is not trivial to make this test more robust using FileCheck and that our main aim with this simple test is just to prove the infrastructure she has added for FileCheck and lit works that it is not worth the trouble of making the test super robust at this point. We should revisit how best to make this test hardened at a later date. We may yet tweak the format of the version output and the new output might be more amenable to robust testing using FileCheck, so we might not hit this issue. We might also consider contributing further changes to FileCheck to make this test easier to write.